stamp_generation/generator_input.py

```diff
@@ -88,8 +88,19 @@ def get_sample_stamp_pages():
                       sentence_index=0,
                       is_embedded_content=False,
                       overlay_title='This is page2 title',
-                      overlay_text='This section is for overlay '
-                                   'text and this is page2',
+                      overlay_text='India has approved the combination of two '
+                      'antiviral drugs used to treat HIV, lopinavir and '
+                      'ritonavir (400 mg and 100 mg, respectively), to '
+                      'treat Covid-19, but the first major study of Covid-19 '
+                      'treatment published in the New England Journal of '
+                      'Medicine (NEJM) found it did not work on severely '
+                      'ill patients in China.\n India has approved the '
+                      'combination of two antiviral drugs used to treat HIV, '
+                      'lopinavir and ritonavir (400 mg and 100 mg, '
+                      'respectively), to treat Covid-19, but the first major '
+                      'study of Covid-19 treatment published in the New '
+                      'England Journal of Medicine (NEJM) found '
+                      'it did not work on severely ill patients in China.',
```

Comment:
Can you move this to statics?

Reply:
I did not understand what you exactly mean by moving it to statics. Can you elaborate?

Reply:
I guess this is for short term. Ignore this comment.

Reply:
I get it. Yes, this script is just for testing purpose which can be removed once end-to-end integration PR is merged.